### PR TITLE
Restruct article to feature-folder

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/WikiService/Controllers/ArticleController.cs [hunk 1]
```diff
@@ -1,41 +1,67 @@
...
+    public async Task<IActionResult> CreateAsync(
+      [FromBody] CreateArticleRequest request,
+      CancellationToken ct)
+    {
+      if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemoveWiki))
+      {
+        return StatusCode(403);
+      }
+
+      return Created("/article", await _mediator.Send(request, ct));
```
Comment (SolovyovDmitry) [inline src/WikiService/Controllers/ArticleController.cs]: I guess resource must be plural

File: src/WikiService/Controllers/ArticleController.cs [hunk 2]
```diff
@@ -1,41 +1,67 @@
...
-      return await command.ExecuteAsync(articleId, request);
+      [FromBody] JsonPatchDocument<EditArticleRequest> request,
+      CancellationToken ct)
+    {
+      if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemoveWiki))
+      {
+        return StatusCode(403);
+      }
+
+      return Ok(await _mediator.Send(articleId, ct));
```
Comment (SolovyovDmitry) [inline src/WikiService/Controllers/ArticleController.cs]: Is request used?

**Review by SolovyovDmitry — APPROVED** [review]

**Comment by SolovyovDmitry** [comment]
I still insist on extracting data and mapping methods so they can be changed easily later without making an impact on handle logic

File: Common.Configurations.targets [hunk 3]
```diff
@@ -28,7 +28,7 @@
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
     <PackageReference Include="MediatR" Version="10.0.1" />
     <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="10.0.1" />
-    <PackageReference Include="DigitalOffice.Kernel" Version="1.0.2" />
+    <PackageReference Include="DigitalOffice.Kernel" Version="1.0.3" />
```
Comment (AnnPostnikova) [inline Common.Configurations.targets]: update kernel version to 1.0.4

**Review by AnnPostnikova — APPROVED** [review]

File: src/WikiService/Controllers/ArticleController.cs (hunk 2 above)
Comment (VLiamin) [reply src/WikiService/Controllers/ArticleController.cs]: done

File: src/WikiService/Controllers/ArticleController.cs (hunk 1 above)
Comment (VLiamin) [reply src/WikiService/Controllers/ArticleController.cs]: done

File: Common.Configurations.targets (hunk 3 above)
Comment (VLiamin) [reply Common.Configurations.targets]: done

**Pull request merged**